djbetty/templatetags/betty.py

```diff
@@ -23,6 +23,9 @@ def __init__(self, id):
     def get_crop_url(self, ratio="original", width=600, format="jpg", fixed=0):
         return self.storage.url(self.id, ratio=ratio, width=width, format=format, fixed=fixed)
 
+    def get_animated_url(self, format="gif", fixed=0):
```

Comment:
Might as well change default `fixed=0` in all places to `fixed=False` (for both animated + cropped methods). Previous author was using `0` for a boolean flag, which is confusing.